tests/acceptance/expected-failures-on-OWNCLOUD-storage.txt

```diff
@@ -1456,7 +1456,6 @@ apiWebdavEtagPropagation1/moveFileFolder.feature:62
 apiWebdavEtagPropagation1/moveFileFolder.feature:78
 apiWebdavEtagPropagation1/moveFileFolder.feature:79
 apiWebdavEtagPropagation1/moveFileFolder.feature:98
-apiWebdavEtagPropagation1/moveFileFolder.feature:99
```

Comment:
This is a surprising change - but CI passed, so I guess that scenario is passing now.